hazelcast/src/hazelcast/client/spi/impl/ClientInvocation.cpp

```diff
@@ -360,55 +373,12 @@ namespace hazelcast {
                 }
 
                 void ClientInvocation::onComplete() {
```

Comment:
This method can be removed. There is no need for it anymore.

Reply:
Removed.